Directory.Build.props

```diff
@@ -1,6 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
```

Comment:
We need to be careful about moving this import up/down in this file, please do a diff of everything in the `artifacts` dir before/after this change to make sure nothing has changed

Reply:
I checked the assembly attributes on a few random files from the bin directory and the nuspec of the generated package and see no change. 

BTW, importing Sdk.props at the top of the file is the [recommended pattern](https://github.com/dotnet/arcade/blob/master/Documentation/ArcadeSdk.md#directorybuildprops) pattern:

> The Sdk.props file sets various properties and item groups to default values. It is recommended to perform any customizations after importing the SDK.

Reply:
@wtgodbe A question - why do you change the `DebugType` to Windows PDBs instead of using the Arcade default which is to generate Portable PDBs?

https://github.com/dotnet/standard/pull/1078/files/9d77bc36f2f360c949118b0955a4c3562e748bdd#diff-4b2f402501d23abbede6eac0f47783e4R87

Reply:
This is an artifact from before we switched to use Arcade in the repo, it's probably safe to change to the default. If you think it's important to do, I can file an issue against myself to make the switch.

Reply:
I did a folder comparison & everything looks good

Reply:
Thanks. Yes, let's remove the [Choose block](https://github.com/dotnet/standard/pull/1078/files/9d77bc36f2f360c949118b0955a4c3562e748bdd#diff-4b2f402501d23abbede6eac0f47783e4R87).